app/models/claim.rb

```diff
@@ -200,7 +200,7 @@ def decision
   end
 
   def decision_made?
-    decision&.persisted?
+    decision.present? && decision.persisted?
```

Comment:
Yeah, this is nicer. I know `nil` is falsey in Ruby, but I prefer `*?` methods to return `true` or `false`